apps/pwabuilder/src/script/pages/app-report.ts

```diff
@@ -2476,8 +2671,8 @@ export class AppReport extends LitElement {
               </div>
 
               <div id="mh-right">
-                ${this.manifestDataLoading ?
-                    html`<sl-skeleton class="progressRingSkeleton" effect="pulse"></sl-skeleton>` :
+                ${this.runningTests ?
```

Comment:
This applies to all three of the cards so I am only going to put this comment once, each card should be full acessible as soon as its respective tests are done running. So if manifest tests are done running, you should have full access to the manifest card including the progress ring, edit your manifest button, and details drop down. This also applies to the SW and Security cards.

Reply:
The action items list should also be fully accessible as soon as the first test completes and action items are available, see production for this.

Reply:
I have unblocked Manifest editor